.github/PULL_REQUEST_TEMPLATE/mod_recipe.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Modified recipe and diagnostic
+about: Modification to an already existing recipe and diagnostic
+title: ''
+labels: 'diagnostic'
+assignees: ''
+
+---
+
+Please discuss your idea for modifications to a diagnostic or recipe with the development team and the author of the diagnostic before getting started, to avoid disappointment later. A good way to do this is to open an issue on GitHub. This is also a good way to get help.
+
```

Comment:
author(s); tell them where they find the autor(s) (in the documentation section of the recipe then check config-references for contact details); encourage GitHub issue by saying: The best way...

Reply:
> Contact information of the author can then be found in esmvaltool/config-references.yml

This is probably not the case, might be better to just tag the author in the github issue

Reply:
@bouweandela I agree and changed this to taggin in the issue